definitions/features/foreman_tasks.rb

```diff
@@ -108,7 +108,7 @@ def fetch_tasks_status(state, spinner)
   end
 
   def services
-    [system_service(service_name, 30)]
+    feature(:dynflow_sidekiq) ? [] : [system_service(service_name, 30)]
```

Comment:
Does that mean rest of things from feature `foreman_tasks` remain as it is and only services get changed?

Reply:
Exactly

Reply:
AFAIK foreman-tasks hasn't had a service ever since dynflowd was introduced. dynflowd is part of Foreman itself while foreman-tasks is an optional plugin. dynflowd.service always had the alias foreman-tasks so `systemctl status foreman-tasks` continued to work.